shell/app/modules/project/pages/test-manage/components/case-tree/index.tsx

```diff
@@ -109,7 +109,7 @@ const TestSet = ({
     caseModal: modalTestSet,
     temp: tempTestSet,
   };
-  const currentTestSet: TEST_SET.TestSet[] = testSet[mode];
+  const currentTestSet: TEST_SET.TestSet[] = React.useMemo(() => testSet[mode], [mode]);
```

Comment:
Pay attention to the lint warning. 
![image](https://user-images.githubusercontent.com/5175455/117908675-6e365a00-b30b-11eb-8bb7-317926f67580.png)